device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/net/IotHubUri.java

```diff
@@ -28,6 +28,9 @@
     /** The API version will be passed as a param in the URI. */
     public static final String API_VERSION = "api-version=" + TransportUtils.IOTHUB_API_VERSION;
 
+    /** The API version will be passed as a param in the URI for PnP. */
+    public static final String PNP_API_VERSION = "api-version=" + TransportUtils.PNP_IOTHUB_API_VERSION;
```

Comment:
I renamed this to API_VERSION_PREVIEW, and added a comment that clarifies that this is for pnp support